gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/jupiter/handlers/api/v4/flow/FlowChain.java

```diff
@@ -81,10 +96,33 @@ public void addHooks(final List<ChainHook> hooks) {
      */
     public Completable execute(ExecutionContext ctx, ExecutionPhase phase) {
         return resolveFlows(ctx)
+            .switchIfEmpty(
+                Flowable.defer(
+                    () -> {
+                        Boolean previousExecutionMatchNoMatch = ctx.getInternalAttribute(CONTEXT_ATTRIBUTES_FLOW_EXECUTION_NO_MATCH);
```

Comment:
**Question:** I can't see where this internalAttribute is set into the context even after reading the PR for APIM-761. Where is it initialized ?

Reply:
just...after it